scss/adminLessonSideNav.module.scss

```diff
@@ -10,35 +10,54 @@ $light-border: hsla(0, 0%, 85%);
 }
 
 .container {
-  background-color: colors.$bg-light-gray;
-  padding: 28px 21px;
-  border-radius: 10px;
   display: flex;
   flex-direction: column;
   align-items: center;
+  padding: 28px 21px;
+  gap: 10px;
+
+  width: 369.01px;
   height: 460px;
-  width: 100%;
-  max-width: 369.01px;
-  row-gap: 23px;
-  justify-content: center;
+
+  background: colors.$secondary;
+
+  box-shadow: 0px 4px 25px rgba(0, 0, 0, 0.07);
+  border-radius: 10px;
 }
 
 .container__heading {
-  font-weight: bold;
+  width: 327px;
+  height: 22px;
+  left: 0px;
+  right: 0px;
+
+  font-weight: 700;
   font-size: 18px;
   font-family: Inter;
   font-weight: 700;
+  font-size: 18px;
+  line-height: 22px;
+  text-align: center;
+  margin-bottom: 15px;
 }
 
 .container__modulesList {
-  padding: 0;
+  padding: 0px;
   flex: 1;
   list-style: none;
   width: 100%;
   display: flex;
   flex-direction: column;
   row-gap: 6px;
   overflow-y: auto;
+
+  font-family: Inter;
+  font-size: 14px;
+
+  width: 327.01px;
+  height: 284px;
```

Comment:
`width` and `height` can't be fixed values because it won't look good on mobile (the page isn't responsive with mobile, but it's a good practice to keep the width responsive).

Before adding these properties, it didn't take an appropriate amount of space?

Reply:
It also looks like there are two width properties.

Reply:
These should be updated now as well